cirq/sim/clifford/clifford_simulator.py

```diff
@@ -37,12 +37,13 @@
 
 import cirq
 from cirq import circuits, study, ops, protocols, value
-from cirq.ops import pauli_gates
 from cirq.ops.clifford_gate import SingleQubitCliffordGate
 from cirq.ops.dense_pauli_string import DensePauliString
-from cirq.protocols import unitary
+from cirq.protocols import act_on, unitary
 from cirq.sim import simulator
-from cirq.sim.clifford import clifford_tableau, stabilizer_state_ch_form
+from cirq.sim.clifford import (act_on_clifford_tableau_args,
```

Comment:
it seems that we could just import the types from these packages in `cirq.sim.clifford.__init__.py` and then use them directly. It is the more typical style in Cirq allowing for shorter names and easier code completion. Then you can do `from cirq.sim import clifford as clifford` and `clifford.ActOn...`

Reply:
You're right, and the style guide agrees with you https://cirq.readthedocs.io/en/latest/docs/dev/style.html.

But I feel I have seen the reverse pattern being followed, the most specific subpaths are imported instead of the top-level module (except in tests where it's `cirq.Whatever`). But I'll at least fix the clifford stuff for this PR to follow the style guide.

Reply:
Had to shuffle the `clifford.__init__.py` to avoid circular dependencies. Since these all classes are very closely intertwined, there is a high chance of circular dependencies if using `cirq.sim.clifford` directly.

Reply:
Yeah now that happens. The init module orderings is a dark art.